tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/AlignedChunkWriterImpl.java

```diff
@@ -186,4 +186,12 @@ public void clearPageWriter() {
       valueChunkWriter.clearPageWriter();
     }
   }
+
+  public TimeChunkWriter getTimeChunkWriter() {
+    return timeChunkWriter;
+  }
+
+  public List<ValueChunkWriter> getValueChunkWriterList() {
+    return valueChunkWriterList;
+  }
```

Comment:
Better not to expose these inner writers to outside. I find that you call these two functions in `checkChunkSize` of `AbstractCompactionWriter`, you can add a method to check whether the size is above threshold instead of exposing these writers to outside.

Reply:
resolved